features/admin_center_manage_display.feature

```diff
@@ -451,3 +451,13 @@ Feature: Manage Display
     And the "Text" field should contain "Updated text for search page alert."
     And the "Status" field should contain "Inactive"
     And I should see "The alert for this site has been updated."
+
+  @javascript
+  Scenario: Editing the Redesigned Display Settings
+    Given the following Affiliates exist:
+      | display_name | name       | contact_email   | first_name | last_name | is_bing_image_search_enabled |
+      | agency site  | agency.gov | john@agency.gov | John       | Bar       | true                         |
+    And I am logged in with email "john@agency.gov"
+    When I go to the agency.gov's Redesigned Display page
+    Then I should see "Redesigned Display"
+    Then I should see a link to "Preview Redesigned Display" with url that ends with "/search?affiliate=agency.gov&redesign=true"
```

Comment:
It would be better to describe the behavior, i.e. 'When I follow "Preview Redesigned Display", Then I should be on agency.gov's Redesigned Display page", but I have a feeling you came to the same conclusion I did: easier said than done. So this is fine for now.